mthree/mitigation.py

```diff
@@ -507,7 +517,13 @@ def apply_correction(
 
         quasi_out = []
         details_out = []
+        log_iter = max(len(counts) // 20, 1)
+        logger.info("Start applying correction using method %s", method)
         for idx, cnts in enumerate(counts):
+            if logger.getEffectiveLevel == logging.DEBUG:
```

Comment:
I normally use this pattern for the condition:

```suggestion
            if logger.isEnabledFor(logging.DEBUG):
```